fpl/fpl.py

```diff
@@ -263,19 +281,25 @@ def __init__(self, session):
         :type return_json: bool
         :rtype: list
         """
-        players = await fetch(self.session, API_URLS["static"])
-        players = players["elements"]
+        players = getattr(self, "elements")
+        gameweek = None
 
         if not player_ids:
-            player_ids = [player["id"] for player in players]
+            player_ids = [player["id"] for player in players.values()]
+
+        if include_live:
+            gameweek = await self.get_gameweek(getattr(self, "current_gameweek"), include_live=True)
 
         tasks = [asyncio.ensure_future(
                  self.get_player(
-                     player_id, players, include_summary, return_json))
+                     player_id, players, gameweek, include_summary, return_json))
                  for player_id in player_ids]
         players = await asyncio.gather(*tasks)
 
-        return players
+        if return_json:
+            return list(filter(lambda p: p["id"] in player_ids, players))
```

Comment:
Can do.

I do think the filter and map functions and nice and concise though, probably clearer
Sometimes an iterator is sufficient rather than a list too

Reply:
I think they are easier to understand, especially for people new to Python.